src/pseudo_branching.jl

```diff
@@ -0,0 +1,207 @@
+struct PSEUDO_COST{BLMO<:BoundedLinearMinimizationOracle} <: Bonobo.AbstractBranchStrategy
+    iterations_until_stable::Int
+    stable::Bool
+    bounded_lmo::BLMO
+end
+
+# """ Function that keeps track of which branching candidates are stable """
+# function is_stable(idx::Int, branching::PSEUDO_COST{BLMO})
+#     local branch_tracker = Dict{Int, Float64}(idx=> 0 for idx in Boscia.get_integer_variables(branching.bounded_lmo))
+#     if branch_tracker[idx] >= branching.iterations_until_stable
+#         return true
+#     else 
+#         return false
+#     end
+# end
+
+
+function pseudo_weight_update!(
+    tree::Bonobo.BnBTree, 
+    node::Bonobo.AbstractNode, 
+    idx::Int,
+    values,
+    pseudos::Dict{Int,Array{Float64}},
+    branch_tracker::Dict{Int, Int},
+    branching::PSEUDO_COST{BLMO}
+) where BLMO <: BoundedLinearMinimizationOracle
+    @assert !isempty(node.active_set)
+    active_set = copy(node.active_set)
```

Comment:
If you are emptying the active set anyway, you can just create an empty one: https://github.com/ZIB-IOL/FrankWolfe.jl/blob/9009acab5d397d71cedee6156977d7d92f79b0b6/src/active_set.jl#L27

The active sets can be come quite large and copying them might lead to some unnecessary overhead.